integration/integration-uiSchema.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { existsSync, readFileSync } from "fs";
+import { ClientFunction, Selector } from "testcafe";
+import { enterPassword, loadConfigFile } from "./helper";
+import { join } from "path";
+import { homedir } from "os";
+import { getData } from "@govtechsg/open-attestation";
+
+fixture("Document Creator").page`http://localhost:3000`;
+
+const Config = "./../src/test/fixtures/sample-local-config.json";
+
+const Title = Selector("h1");
+const Button = Selector("button");
+const Form = Selector("[data-testid='form-group field field-object']");
+const AddNewButton = Selector("[data-testid='add-new-button']");
+const ProgressBar = Selector("[data-testid='progress-bar']");
+const SubmitButton = Selector("[data-testid='form-submit-button']");
+const DocumentTitleField = Selector("#root_title");
+const DocumentRemarksField = Selector("#root_remarks");
+const goBack = ClientFunction(() => window.history.back());
+const DownloadLink = Selector("[data-testid='download-file-button']");
+
+function getFileDownloadPath(fileName: string): string {
+  return join(homedir(), "Downloads", fileName);
```

Comment:
@yehjxraymond I've since changed to use the `path.join` and `os.homedir` method to get the `Downloads` folder

We can still do a simulated download using this method, except that we don't have to download xdg-user-dirs package